book/src/faq.md

```diff
@@ -96,20 +96,22 @@ when a change is signaled, primarily to minimize latency and optimize client req
 By default, if your service does not emit change notifications, the property values will not
 update accordingly. 
 
+
 However, you can disabling caching for specific properties:
 
 - Add the `#[dbus_proxy(property(emits_changed_signal = "false"))]` annotation to the property
-for which you desire to disable caching on.
+for which you desire to disable caching on. 
+
+> For more information about all the possible values for `emits_changed_signal` refer
+ to [`dbus_proxy`] documentation.
```

Comment:
I really don't know how you got the idea that I was asking for block quotes. :) I did not. I only asked for the paragrpah to be removed and sentence to be just appended to this bullet point (no need for separate paragraph).

Reply:
Yup you never said to but I just thought it looked a little better this way. No problem though I'll change it